src/components/Insertion/Results/index.tsx

```diff
@@ -1,29 +1,35 @@
-import { collection, getDocs } from 'firebase/firestore';
 import { useEffect, useState } from 'react';
 import { useSearchParams } from 'react-router-dom';
-import { db } from '../../../firebaseConfig';
 import { InsertionData } from '../types';
+import { searchClient } from '@algolia/client-search';
+
+const appID = import.meta.env.ANGOLIA_APP_ID;
```

Comment:
mi soprende funzioni? non serve prefissarle con `VITE_`? magari verifichiamo in preview su netlify

Reply:
Stava funzionando, comunque per sicurezza faccio un refactor